SciPy2024/module1/introduction-to-language-models.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
<p>Neural models could be described as special type of statistical models. Instead, maybe focus on some other aspect like interpretability or amount of data used for training.<em> </em>For instance - Co<em>mmercially integrated LLMs use neural networks for training on large amounts of data whereas statistical models are more interpretable and better suited for smaller datasets</em> Or what type of tasks each type is better suited to. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-ssec/tutorials/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>